rollup.config.ts

```diff
@@ -26,7 +26,7 @@ const buildCjsPackage = ({ env }) => {
         },
       },
     ],
-    external: ['react', '@grafana/data', '@grafana/ui', 'lodash'],
+    external: ['react', '@grafana/data', '@grafana/ui', '@grafana/runtime', 'lodash', 'rxjs'],
```

Comment:
The reason this grafana/aws-sdk did not use grafana/runtime previously was because before Grafana 9.0, grafana/ui depended on grafana/aws-sdk. This created an implicit dependency between grafana/ui and grafana/runtime and that is really bad. 

This dependency was removed in 9.0, so this should be totally fine for any user running that version or newer versions of Grafana. However, I believe AMG are still running 8.4.x, so I think the long running queries feature must be compatible with that version. Just to make sure I'm correct on this, can you try and build all repos involded in the long running queries feature together with Grafana 8.4.x? If this doesn't work, we may have to break out the AsyncDataSourceWithBackend.ts (that's the only file who needs the runtime, no?) into a new "AsyncDataSourceBackend" repo/npm package. That wouldn't be so bad anyway because the long term intent is that this feature is made available for wider use beyond just AWS plugins.

Reply:
I pulled down Grafana v8.4.7 to test and I didn't see any errors.

Reply:
Okay good, thanks for testing!